demos/gallery/samples/layout/VerticalList.js

```diff
@@ -10,7 +10,6 @@ const data = [
     ["F", 144, 233]
 ];
 new VerticalList()
-    .target("target")
     .itemMinWidth(85)
     .itemMinHeight(68)
```

Comment:
Are these not PP?

Reply:
They are, but I assumed the publish params being above or below the `.target("target")` didn't really matter. I considered just search+replace:
`".render("` with ".target('target').render(". Would that be easier to merge? TravisCI should be able to catch the samples that would break after that mass string replace.

Reply:
When you change the props in the UI it will attempt to update the props in the source code (but only between the target and render)